indico/modules/rb_new/client/js/common/user/reducers.js

```diff
@@ -60,6 +66,7 @@ export default combineReducers({
                     avatarBgColor: user.avatar_bg_color,
                     language: user.language,
                     isAdmin: user.is_admin,
+                    isRBAdmin: user.is_rb_admin,
```

Comment:
Can't we just do sth like this `return camelizeKeys(action.data.user)`?

Reply:
I think that'd give us `isRbAdmin` here